core/translation.cpp

```diff
@@ -904,6 +908,8 @@ void Translation::_bind_methods() {
 	ClassDB::bind_method(D_METHOD("_set_messages"), &Translation::_set_messages);
 	ClassDB::bind_method(D_METHOD("_get_messages"), &Translation::_get_messages);
 
+	BIND_VMETHOD(MethodInfo(Variant::String, "_get_message", PropertyInfo(Variant::STRING, "src_text")));
```

Comment:
```suggestion
	BIND_VMETHOD(MethodInfo(Variant::STRING, "_get_message", PropertyInfo(Variant::STRING, "src_message")));
```

Reply:
You should also add it to the documentation in `doc/classes/Translation.xml`, and write the docs. See https://docs.godotengine.org/en/latest/community/contributing/updating_the_class_reference.html

Reply:
I've tried to update XML and doctool generated:
```xml
		<method name="_get_message" qualifiers="virtual">
			<return type="String" />
			<argument index="0" name="src_message" type="String" />
			<description>
			</description>
		</method
```

@akien-mga Should it be marked as `const` as well?

Reply:
No, if it's not done by doctool, it shouldn't be added manually (it would be removed by doctool).